### DISCUSSION TITLE
Unable to find problem caused by component naming

### DISCUSSION THREAD

**Discussion by weeptao** [vercel/next.js#69207] [Help]
### Summary

When I use component<A.b/>, there may be a bug in the React Server Components bundle
How to use such component naming
![image](https://github.com/user-attachments/assets/7ac49423-b75b-4831-a78e-662f0bf1fed8)

**Comment by suprunchuk** [comment]
Provide a more detailed demo project to reproduce your problem.

---

Example of a Proper Component Import and Usage:
```jsx
import { A } from './components/AModule';

const MyComponent = () => {
  return (
    <div>
      <A.b />
    </div>
  );
};

export default MyComponent;
```
